cmd/loadgenerator/main.go

```diff
@@ -141,14 +211,11 @@ func waitForRound(restClient client.RestClient, cfg config, spendingRound bool)
 	}
 }
 
-func generateTransactions(restClient client.RestClient, cfg config, privateKey *crypto.SignatureSecrets, publicKey basics.Address) (queueFull bool) {
-	var nodeStatus generatedV2.NodeStatusResponse
+const transactionBlockSize = 800
+
+func generateTransactions(restClient client.RestClient, cfg config, privateKey *crypto.SignatureSecrets, publicKey basics.Address, nodeStatus generatedV2.NodeStatusResponse) (queueFull bool) {
+	start := time.Now()
 	var err error
-	nodeStatus, err = restClient.Status()
```

Comment:
oh OK without this change, multiple loadgenerators wouldn't necessarily all target the same block... but one might also miss the block (like we were seeing) if something slow happens between waitForRound and generateTransactions?

Reply:
I can't think of any reason why you would miss the round by the time you get around to sending, unless the node you're talking to was running behind or something